.kokoro/continuous/bigtable-it.cfg

```diff
@@ -8,7 +8,7 @@ env_vars: {
 
 env_vars: {
     key: "INTEGRATION_TEST_ARGS"
-    value: "google-cloud-clients/google-cloud-bigtable -Dbigtable.env=prod -Dbigtable.project=gcloud-devel -Dbigtable.instance=google-cloud-bigtable -Dbigtable.table=integration-tests"
+    value: "google-cloud-clients/google-cloud-bigtable,google-cloud-examples -Dbigtable.env=prod -Dbigtable.project=gcloud-devel -Dbigtable.instance=google-cloud-bigtable -Dbigtable.table=integration-tests -PbigtableITTest"
```

Comment:
This isn't working right now. See the [log](https://source.cloud.google.com/results/invocations/1e8d0d6b-489d-479e-a9b9-10c4e5ed778f/targets/cloud-devrel%2Fclient-libraries%2Fjava%2Fgoogle-cloud-java%2Fpresubmit%2Fbigtable-it/log)

The `.kokoro/build.sh` script will need to be modified to split the first integration argument and handle a list of paths to:
* check for changes in any of the split directories
* run the tests in each of the split directories